custom_model_runner/requirements.txt

```diff
@@ -15,7 +15,7 @@ scipy>=1.1,<2
 strictyaml==1.4.2
 texttable
 py4j~=0.10.9.0
-pyarrow==2.0.0
+pyarrow
```

Comment:
why did you unpin pyarrow

Reply:
We have different pyarrow versions pinned in the DRUM an in environments. This way there won't be version conflict.

Reply:
what happens if there is a conflict?

Reply:
We use base envs which have older pip, which doesn't scream if there is conflict.
Now I created new virtual env, and there is a conflict. That's why I unpinned version in drum.

Reply:
I think we should leave it pinned. I could see this breaking things in the future

Reply:
So I can pin it and degrade pip not to brake on conflicts.
Any how this is only temp measure, all this will be fixed as we need to create tests/envs for Py3.9